### DISCUSSION TITLE
Where can I find a complete list of all entities that can be detected by GLiNER?

### DISCUSSION THREAD

**Discussion by aramfaghfouri** [urchade/GLiNER#59] [Q&A]
Hi,

Thank you for this great work.
Could you please let me know where I can find a complete list of all entities that can be detected by GLiNER?

**Comment by urchade** [comment]
GLiNER has been trained to detect any types of entities. However, it does not perform evenly across domains.

**Reply by aramfaghfouri** [reply]
Thank you. Is there any list that you could share as a reference for correctly naming and listing all possible entities?

**Comment by DavidFarago** [comment]
Such a list would be very helpful. For instance, to detect legal references in German text, I tried various labels like "law", "legal reference", "juristic document", and ended up using "§". I also wonder whether labels should always be English. Lot's of guesswork that would become easier seeing the complete list of all entity types (labels) occurring in the training data.
